packages/server/src/database.ts

```diff
@@ -11,6 +19,24 @@ export enum DatabaseMode {
 
 let pool: Pool | undefined;
 let readonlyPool: Pool | undefined;
+let dataVersion = -1;
+let pendingDataMigration = -1;
+
+export function getCurrentDataVersion(): number {
+  return dataVersion;
+}
+
+export function getPendingDataMigration(): number {
+  return pendingDataMigration;
+}
+
+export async function markPendingDataMigrationCompleted(job: AsyncJob): Promise<void> {
+  assert(job.dataVersion);
+  await getDatabasePool(DatabaseMode.WRITER).query('UPDATE "DatabaseMigration" SET "dataVersion" = $1', [
+    job.dataVersion,
+  ]);
+  pendingDataMigration = 0;
```

Comment:
Why 0 here and not -1?

Reply:
Making specific `const` values for -1 and 0 might be useful, at least

Reply:
Oh good call